o-fish-ios/Views/PatrolBoat/PatrolBoatView.swift

```diff
@@ -211,6 +211,7 @@ struct PatrolBoatView: View {
     private func logoutAlertClicked() {
         RealmConnection.logout()
         isLoggedIn.wrappedValue = false
+        NotificationManager.shared.removeAllNotification()
```

Comment:
is this methods remove all the pending notifications?

Reply:
This methods removes all notification: pending requests and delivered notifications.